src/pykiso/cli.py

```diff
@@ -173,6 +173,11 @@ def get_logging_options() -> LogOptions:
     default=None,
     help="allow the user to execute a subset of tests based on branch levels",
 )
+@click.option(
+    "--fail-fast",
```

Comment:
Maybe add the option flag in the same format as unittest (-f, --failfast)

Reply:
as you want